internal/expr/parts.go

```diff
@@ -42,6 +42,17 @@ func (p *inputPart) String() string {
 
 func (p *inputPart) part() {}
 
+type inPart struct {
+	types []fullName
```

Comment:
How do we handle `foo IN ("one", $Foo.two)`, and why is `IN` special in that regard, when in general we're happy to inject the values blindly? Wouldn't it be simpler if we just let the value through?

I'm missing the design conversation for this feature. Do we have a spec for it?

Reply:
See update comment below